baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -142,8 +156,9 @@ def __init__(
         :param queue_name: Name for your queue.
         :param routing_keys: List of routing keys that you will create :py:class:`~kombu.Queue` s
             to consume from.
-        :param handler_fn: A `baseplate.frameworks.queue_consumer.komub.Handler`
-            function that will process an individual message from a queue.
+        :param handler_fn: A function that will process an individual message from a queue.
+        :param error_handler_fn: A function that will be called when an error is thrown
+            while executing the `handler_fn`.
```

Comment:
Good point about why we need this, can you update this comment to reflect that `message.ack` and/or `message.requeue` will _not_ be called automatically if this is set when an error is raised and you need to ensure that this function handles that?